.github/workflows/ci.yml

```diff
@@ -138,6 +138,7 @@ jobs:
           set -e
 
           # build for WebGPU
+          cargo clippy --target ${{ matrix.target }} --tests --features glsl,spirv,fragile-send-sync-non-atomic-wasm
```

Comment:
I would like to add testing with `target-feature=+atomics` as well, but that would require Rust nightly.
Should I add another matrix entry for that?

Reply:
I honestly wouldn't bother on it right now, we need a central place to document this, likely in the readme.

Reply:
Alright.